src/disco/components/Addon/index.js

```diff
@@ -226,13 +227,13 @@ export class AddonBase extends React.Component {
   };
 
   clickHeadingLink = (e) => {
-    const { type, name, _tracking } = this.props;
+    const { addon, _tracking } = this.props;
```

Comment:
is this still needed?

Reply:
why not? it is needed for the tracking event.

Reply:
ok was just verifying..I wasn't sure if I ever saw this called

Reply:
ohhh! I see...it's just tracking the click : )